src/ListView/PageTableHoverOptions.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { HoverDisclosure, OptionsMenu } from "@sailthru/stui-components";
+
+const PageTableHoverOptions = ({ pageId, pageEditorUrl }) => {
+  const clickOptions = [
+    {
+      label: "Edit",
+      icon: "fal fa-edit fa-fw",
+      iconHover: "fas fa-edit fa-fw",
+      onClick: () => window.open(`${pageEditorUrl}`, "_self"), // TODO see EPT-1713 (add routing?)
+    },
+    {
+      label: "Delete",
+      icon: "fal fa-trash fa-fw",
+      iconHover: "fal fa-trash fa-fw",
```

Comment:
Does this need to be updated to make the hover a bolder icon?
```suggestion
      iconHover: "fas fa-trash fa-fw",
```

Reply:
Yes! updated